app/controllers/application_controller.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  private
+
+  def user_registration_url_method
+    method(:new_user_registration_url)
```

Comment:
@Vladimir-Kuchinskiy why we can't call this method directly?

Reply:
Actually we can do this and now with new link format, it should be better to call this method from the controller, and pass the /users/sign_up link to service.